05-how-to-contribute-to-open-source.md

```diff
@@ -3,7 +3,11 @@
 In this chapter, we'll guide you through the process of contributing to open source projects. We'll cover the following topics:
 
 - Finding projects to work on
+- Getting started with contributing
 - How to submit a contribution
+- Let's get practical
+- Keeping branches up to date
+- Merge conflicts
 - Onboarding in a new project
 - What happens next?
```

Comment:
I think these can be hyperlinked instead @adiati98. It'll make the navigation of this chapter easier.  What do you think @BekahHW?

Reply:
If we want to make this a table of contents, then we might as well want to add a table of contents to all chapters in the course.

Reply:
As this is a different thing, I won't work on it in this PR.
If we agree to whatever the outcome, let's create a new issue to fix this. :)

Reply:
I think you're right, @adiati98  bc they're on the sidebar, let's remove them.

Reply:
I was thinking to create a new issue for this if we want to add a TOC because then we'll need to add TOC to all chapters. But if we want to remove them, should I just do it in this PR, @BekahHW?

Reply:
Issue #101